haskell_101/codelab/01_functions/setup.md

```diff
@@ -0,0 +1,16 @@
+============================================
+
+             HASKELL MTV 101
+
+============================================
+
+If you are reading this text, probably you don't have Haskell environment setup
+on your system. This should give you enough instructions to install it.
+
+First, note that there are two different ways to get Haskell installed:
+
+1. using [Stack](https://docs.haskellstack.org/en/stable/README/)
+2. getting the [Haskell Platform](https://www.haskell.org/platform/)
+
+This course is not taking any side, you are free to install either of these. The
+links above point to the install pages for either of the toolchains.
```

Comment:
I wonder if it is still relevant to suggest both options here.
https://www.haskell.org/platform/ redirects to https://www.haskell.org/downloads/ for me, and then `stack` is there as a second option.

If someone is running on Windows, it does contain instruction for Windows installation.
I have no idea how `stack` behaves on Windows.
But if it works there as well, maybe just suggest `stack`?
Otherwise, suggest `stack`, unless someone is on Windows, and then suggest `https://www.haskell.org/downloads/#windows`?

Reply:
I think it's still better to not take any sides. Maybe in the future we can suggest `ghcup` which is supposed to work even better and manage all installation formats, but need to first test this sometimes.